releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,14 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 4.0.0-preview - July 23, 2020
+* Added `IConfigurationRefresher.SetDirty` method  to set the cached value for key-values registered for refresh as dirty. For users that use push notifications from event grid to receive configuration change updates, this allows the cached values to be revalidated on the next call to `RefreshAsync` or `TryRefreshAsync`.
+* Added support for deserializing configuration settings with JSON content type. [#191](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/191)
```

Comment:
How about this?

Added JSON content-type (eg., MIME type `application/json`) support for key-values in App Configuration, so primitive types, arrays, and JSON objects can be loaded to `IConfiguration` properly.

Reply:
Sounds good.